source/deventsystem/eventdispacher.d

```diff
@@ -59,7 +59,7 @@ public:
 	 */
 	@system
 	void dispach(T : Event)(in void delegate(T) func)
-		in(func.ptr !is null)
+		in(func.ptr !is null || func.funcptr !is null)
```

Comment:
You may want `&&` operator here, right?

Reply:
No, you're passing a delegate for example, `func.ptr is null` however `func.funcptr is not null`. If at least one of them is not null, then func is valid.